test/acceptance/timeout.js

```diff
@@ -18,10 +18,21 @@ describe('timeouts', function(){
     }, 300);
   })
 
+  describe('enabling', function() {
+    this.timeout(0);
+
+    it('should allow overriding per-test', function(done) {
+      // uncomment
```

Comment:
But then the test will fail because the timeout will throw.

I used the same technique from here: https://github.com/shaharmor/mocha/blob/fix/2051-runnable-timeout/test/acceptance/timeout.js#L3